docs/mp/integration/wls.adoc

```diff
@@ -0,0 +1,707 @@
+:wls-jaxrs-client-link: {wls-jaxrs-url}[Developing RESTful Web Service Clients for Oracle WebLogic Server]
+
+:wls-jms-url: {wls-root-url}/wlshe/integrating-oracle-weblogic-server-jms-helidon.html#GUID-8E1A4C48-D7C9-4599-B0C1-89E1FEA823B2
+:wls-jms-link: {wls-jms-url}[Integrating WebLogic JMS with Helidon]
+
+:wls-thin-url: {wls-root-url}/saclt/wlthint3client.html#GUID-D352B8F0-1B9D-43E8-A675-BBFE7E4DE1E9
+:wls-thin-link: {wls-thin-url}[Weblogic thin T3 client]
+
+:wls-soap-url: {wls-root-url}/wlshe/integrating-oracle-weblogic-server-web-services-helidon.html#GUID-DA893D15-13CB-49AF-AD5B-43FD716618BB
+:wls-soap-link: {wls-soap-url}[Integrating Oracle WebLogic Server Web Services with Helidon]
+
+:wls-sso-url: {wls-root-url}/wlshe/integrating-oracle-weblogic-and-helidon-sso-oci.html#GUID-D54EE368-0D7F-4DBB-9279-CC5BDDB76725
+:wls-sso-link: {wls-sso-url}[Integrating Oracle WebLogic Cluster and Helidon Applications for SSO on OCI Using IDCS]
+
+:wls-xa-url: {wls-root-url}/wlshe/integrating-oracle-weblogic-server-microtx.html#GUID-81107BDE-7A99-421B-883D-AAAC12692E26
+:wls-xa-link: {wls-sso-url}[Integrating XA Global Transactions Between WebLogic Server and Helidon Using MicroTx]
+
+:jersey-url: https://eclipse-ee4j.github.io/jersey.github.io/documentation/latest
+:jersey-link: {jersey-url}[Eclipse Jersey]
+
+:micro-tx-url: https://www.oracle.com/database/transaction-manager-for-microservices
+:micro-tx-link: {micro-tx-url}[MicroTx]
+
+== Contents
+
+- <<Overview, Overview>>
+- <<REST Services Integration, REST Services Integration>>
+- <<JMS Integration, JMS Integration>>
+- <<Web Services Integration, Web Services Integration>>
+- <<Single Sign-On on OCI Integration, Single Sign-On on OCI Integration>>
+- <<XA Transactions Integration, XA-Transactions-Integration>>
+
+
+== Overview
+
+The Oracle WebLogic Server (WebLogic Server) and Helidon integration enables interaction between a Helidon microservice application and an application installed on WebLogic Server.
+
+The following integrations are explained in this document:
+
+* The REST integration allows WebLogic Server applications and Helidon microservices to communicate through RESTful Web Service invocations.
+* The JMS integration allows Helidon microservices to publish and consume messages from WebLogic JMS Server.
+* The WebLogic Web Services integration allows Helidon microservices to interact with WebLogic Server applications through SOAP (Simple Object Access Protocol) Web Service calls from Helidon to WebLogic Server.
+* The WebLogic Server and Helidon integration allows communication between a WebLogic cluster-hosted application and a Helidon microservice application by implementing Single Sign-on (SSO) authentication using Oracle Identity Cloud Service (IDCS).
+* XA Transactions Integration with {micro-tx-link} allows participating in XA transactions to both Helidon and WebLogic Server applications.
```

Comment:
"XA Transactions Integration with {micro-tx-link} allows XA transactions to span Helidon and WebLogic Server applications."

Reply:
Updated to "XA Transactions integration with {micro-tx-link} allows Helidon and WebLogic Server applications to participate in a two-phase commit."